sscanss/app/commands/insert.py

```diff
@@ -133,33 +133,34 @@ class InsertTomographyFromFile(QtWidgets.QUndoCommand):
     :param sizes_and_centres: array of pixel positions (if loading a stack of TIFF's) or empty (if loading a nexus file)
```

Comment:
You missed this one. Also I think the new argument should be called `volume_centre`  instead of `pixel_centres`.